config/parse_archive.go

```diff
@@ -31,6 +31,14 @@ func ParseArchive(archiveData []byte, ext string) (*InternalTfConfig, error) {
 			// We skip nested files when parsing manifest
 			return nil
 		}
+		if strings.HasSuffix(strings.ToLower(filename), "readme.md") {
```

Comment:
It might be better to check if the filename is exactly `readme.md`. Could be weird if someone adds something like `another-readme.md` and it picks it up.

Reply:
Fixed